manifests/kustomize/env/aws/accesskey/aws-configurations-patch.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: workflow-controller-configmap
+  namespace: kubeflow
+data:
+  config: |
+    {
+    executorImage: $(executorImage),
+    artifactRepository:
+    {
+        archiveLogs: true,
+        s3: {
+            bucket: $(artifactRepositoryBucket),
+            keyPrefix: $(artifactRepositoryKeyPrefix),
+            endpoint: s3.amazonaws.com,
+            insecure: false,
+            accessKeySecret: {
+                name: $(awsSecretName),
+                key: $(awsAccessKeySecretKey)
+            },
+            secretKeySecret: {
+                name: $(awsSecretName),
+                key: $(awsSecretKeySecretKey)
+            }
+        }
+    }
+    }
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: ml-pipeline-ui
+spec:
+  template:
+    spec:
+      containers:
+        - name: ml-pipeline-ui
+          env:
+            - name: AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: $(awsSecretName)
+                  key: $(awsAccessKeySecretKey)
+            - name: AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: $(awsSecretName)
+                  key: $(awsSecretKeySecretKey)
+            - name: ARGO_ARCHIVE_LOGS
+              value: "true"
+            - name: ARGO_ARCHIVE_ARTIFACTORY
```

Comment:
@Bobgy @IronPan  I wonder why UI is not getting these values from ConfigMap. Probably the configmap did not exist at the time.